en/flight_modes_vtol/README.md

```diff
@@ -0,0 +1,20 @@
+# Flight Modes (VTOL)

```

Comment:
This introduction is somewhat better than before, since all we did was say that VTOL uses MC modes and FW modes based on its flying config. Now we say that, but point to the main exception docs.

However it is not as good if you're a VTOL user as say an MC, where you get a full list of the modes.

I think good enough for now though, since if you've started working in VTOL you probably already know about the other mode types.